packages/api/bin/cli.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/* eslint-disable no-console */
+
+const pckg = require('../package.json');
+const es = require('./es');
+const program = require('commander');
+const { cliUtils } = require('@cumulus/common');
+
+program.version(pckg.version);
+
+program
+  .usage('TYPE COMMAND [options]');
+
+program
+  .command('reindex')
+  .description('Reindex elasticsearch index to a new destination index')
+  .option('-a, --index-alias <alias>', 'AWS Elasticsearch index alias', 'cumulus-alias')
+  .option('--host <host>', 'AWS Elasticsearch host', null)
+  .option('-s, --source <sourceIndex>', 'Index to reindex', null)
+  .option('-d, --dest-index <destIndex>',
+    // eslint-disable-next-line max-len
+    'Name of the destination index, should not be an existing index. Will default to an index named with today\'s date',
+    null)
+  .action(async (cmd) => {
+    if (cliUtils.verifyRequiredarameters([{ name: 'host', value: cmd.host }])) {
+      try {
+        const response = await es.reindex(
+          cmd.host,
+          cmd.sourceIndex,
+          cmd.destIndex,
+          cmd.alias
+        );
+
+        console.log(response);
+      }
+      catch (err) {
+        console.log(err.message);
```

Comment:
I would probably make this `console.error` so that it prints to STDERR instead of STDOUT.